src/current/v23.1/cpu-profiler.md

```diff
@@ -0,0 +1,71 @@
+---
+title: CPU Profiler
+summary: Verify that your cluster's data replication, data placement, and zone configurations are working as expected.
```

Comment:
Nit: this summary does not match what this page is about.

Reply:
Updated Summary.